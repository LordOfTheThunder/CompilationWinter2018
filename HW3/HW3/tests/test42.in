void StructInsideFunc() {
	struct potato {
		int x;
		bool y;
		byte z;
	};
	struct potato s;
	s.x = 3;
	s.y = false;
	s.z = 31b;
	s.x = s.z;
}
