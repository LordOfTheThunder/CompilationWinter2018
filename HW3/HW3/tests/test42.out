---end scope---
struct potato {INT x,BOOL y,BYTE z}
s struct potato 0
---end scope---
print (STRING)->VOID 1
printi (INT)->VOID 2
StructInsideFunc ()->VOID 3
